pkg/proxy/client/client_conn.go

```diff
@@ -76,7 +81,9 @@ func (cc *ClientConnection) Run(ctx context.Context) {
 		return
 	}
 
-	if err := cc.processMsg(ctx); err != nil {
+	if err := cc.processMsg(ctx); err != nil &&
+		// ignore EOF from client
+		!(errors.Is(err, ErrClientConn) && errors.Is(err, io.EOF)) {
```

Comment:
There may occur various disconnection errors. Maybe using `IsDisconnectError` is better.

Reply:
Done.